platform/platform-scheduler/src/main/java/ddf/platform/scheduler/CommandJob.java

```diff
@@ -53,132 +55,132 @@ protected Subject getSystemSubject() {
   }
 
   @Override
-  public void execute(final JobExecutionContext context) throws JobExecutionException {
-    SECURITY.runAsAdmin(
-        () -> {
-          Subject subject = getSystemSubject();
-
-          if (subject != null) {
-            subject.execute(
-                () -> {
-                  doExecute(context);
-                  return null;
-                });
-          } else {
-            LOGGER.debug("Could not execute command. Could not get subject to run command");
-          }
-
-          return null;
-        });
+  public void execute(final JobExecutionContext context) {
+    try {
+      SECURITY.runAsAdmin(
+          () -> {
+            final String commandString = getCommandString(context);
```

Comment:
❓ Should we move this line before the `try` block and use the `commandString` in the `catch (RuntimeException)` log messages? That would help admins know which command failed to run.